client/galaxy/scripts/components/Workflow/Editor/WorkflowRecommendationsVue.vue

```diff
@@ -0,0 +1,125 @@
+        <div class="modal-backdrop fade in"></div>
+        <div class="modal-dialog wf-tools-dialog">
+            <div class="modal-content wf-tool-content">
+                <div class="modal-header wf-tools-background" :title="modalHeaderToolTip">
+                    <h4>{{ modalHeaderTitle }}</h4>
+                </div>
+                <div class="modal-body wf-tools-body">
+                    <div v-if="showLoading">
+                        {{ loadingMessage }}
+                    </div>
+                    <div v-if="compatibleTools.length > 0 && !isDeprecated">
+                        <div v-for="tool in compatibleTools">
+                            <i class="fa mr-1 fa-wrench"></i>
+                            <a href="#" title="Open tool" :id="tool.id" v-on:click="createTool(tool.id)">
+                                {{ tool.name }}
+                            </a>
+                        </div>
+                    </div>
+                    <div v-else-if="isDeprecated">
+                        {{ deprecatedMessage }}
+                    </div>
+                    <div v-if="compatibleTools.length === 0 && !showLoading">
+                        {{ noRecommendationsMessage }}
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <div class="buttons">
+                        <button v-on:click="closeModal" title="Cancel" class="wf-tools-background">Cancel</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import { getAppRoot } from "onload/loadConfig";
+import axios from "axios";
+import { getModule } from "./services";
+import _l from "utils/localization";
+
+export default {
+    props: {
+        workflowManager: {
+            type: Object
+        },
+        toolSequence: {
+            type: String
+        }
+    },
+    data() {
+        return {
+            modalHeaderTitle: _l("Tool Recommendations"),
+            modalHeaderToolTip: _l(
+                "The recommended tools are shown in the decreasing order of their scores predicted using machine learning analysis on workflows. A tool with a higher score (closer to 100%) may fit better as the following tool than a tool with a lower score. Please click on one of the following/recommended tools to have it on the workflow editor."
+            ),
+            compatibleTools: [],
+            isDeprecated: false,
+            noRecommendationsMessage: _l("No tool recommendations"),
+            deprecatedMessage: "",
+            loadingMessage: _l("Loading recommendations ..."),
+            showLoading: true
+        };
+    },
+    created() {
+        this.loadRecommendations();
+    },
+    computed: {},
+    methods: {
+        async loadRecommendations() {
+            const response = await axios.post(`${getAppRoot()}api/workflows/get_tool_predictions`, {
```

Comment:
Axios calls should be placed into the service.js file.

Reply:
ok, will do it